### PR TITLE
Preparation for version v2017.4.2

### PR DESCRIPTION
**Pull request by matsduf**
Preparation for version v2017.4.2
* Update Changes.
* Update README.md.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,22 @@
 Release history of Zonemaster
 
+v2017.4.2 2018-02-23 Security release
+
+Changes:
+
+- Security release of zonemaster-gui
+- Update of validation in zonemaster-backend
```
Comment (sandoche2k) [inline Changes]: Update of domain input validation in zonemaster-backend

File: Changes [hunk 2]
```diff
@@ -1,5 +1,22 @@
 Release history of Zonemaster
 
+v2017.4.2 2018-02-23 Security release
+
```
Comment (sandoche2k) [inline Changes]: Shouldn't it be v2018.1?

As per https://github.com/dotse/zonemaster/blob/b461694b841121b2a0bf9a3b847fdb8d72f10857/docs/design/Versions%20and%20Releases.md
"Version Number Syntax for Zonemaster product"

Comment (matsduf) [reply Changes]: This is an update outside the plan. We have only patching of two components. We could make this v2018.1, but then we have to change the plan too.

Both solutions are OK for me.

**Comment by sandoche2k** [comment]
@matsduf i approve for 2017.4.2

**Pull request merged**